### DISCUSSION TITLE
webrtcICEHostNAT1To1IPs list format

### DISCUSSION THREAD

**Discussion by bryandario8** [bluenviron/mediamtx#2524] [Questions]
### Question

The documentation indicates that webrtcICEHostNAT1To1IPs may contain a list of ips, but it only works for me with one ip, e.g. [127.0.0.1]. I want to configure two or more hosts, is it possible?

Using MediaMTX 1.0.0

**Comment by aler9** [comment]
Hello, the issue you described is a limitation that has been superseded in MediaMTX 1.3.0: `webrtcICEHostNAT1To1IPs` has been replaced with `webrtcAdditionalHosts`, that supports multiple IPs and even DNS names. Instructions are here: https://github.com/bluenviron/mediamtx#connectivity-issues

[a comment by an automated account is not shown]
